streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/HighAvailabilityTaskAssignorTest.java

```diff
@@ -97,6 +100,7 @@
 import static org.mockito.Mockito.spy;
 
 @RunWith(Parameterized.class)
+@Category({IntegrationTest.class})
```

Comment:
I looked into the 90 day history of this test, and the last failure was on Dec/11 the day we merged https://github.com/apache/kafka/pull/14965

Hence, this test should be stable now, and I don't think we need to add this tag, because it's really a unit test after all.

Reply:
The average duration for this suite is ~4m30s, which is the 44th longest test suite in the project. This is borderline, but fell on the high side of the arbitrary line I chose.